src/css/profile/mobile/common/popup.less

```diff
@@ -187,8 +186,8 @@
 			content: "";
 			position: absolute;
 			width: @px_base;
-			height: 22 * @px_base;
-			top: 7 * @px_base;
+			height: 16 * @px_base;
+			top: 10 * @px_base;
 			background-color: var(--on-off-switch-divider-color);
 			opacity: var(--on-off-switch-divider-opacity);
```

Comment:
Need to change the opacity to use the number between 0 and 1.

Reply:
I will change it after the meeting today.